public/js/controllers/charts_controller.js

```diff
@@ -434,66 +453,69 @@ export default class extends Controller {
     this.lastZoom = Zoom.object(start, end)
     this.settings.zoom = Zoom.encode(this.lastZoom)
     this.query.replace(this.settings)
-    this.setSelectedZoom(Zoom.mapKey(this.lastZoom, this.limits))
   }
 
-  setSelectedZoom (zoomKey) {
```

Comment:
The two different functions `onZoom` and `setSelectedZoom` were for two different purposes. One just set the selected button, one was a button callback which set `this.settings.zoom` and redrew the chart. They can't be combined in the way that you've done here. The zoom selector is no longer working.

![image](https://user-images.githubusercontent.com/6109680/58119594-f5dba900-7bc8-11e9-85a4-0606a64da078.png)